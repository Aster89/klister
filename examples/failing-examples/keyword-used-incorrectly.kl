#lang "prelude.kl"

(import "keyword-test.kl")

(example
  (my-then "math works"
   my-else "math is broken"))
