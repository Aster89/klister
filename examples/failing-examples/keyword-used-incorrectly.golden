Syntax error from macro:
    #[keyword.kl:8.20-8.62]<(Keyword my-then cannot appear on its own)>
